content/4. Deeltaken/3. Processen/4. Implementatieproces/2. Bouwen softwaresysteem/Boolean/1. Boolean.md

```diff
@@ -0,0 +1,120 @@
+---
+title:
+  - 1. Boolean
+taxonomie:
+  - ib-19.2.Datatype-Primitief-Boolean.DT
+  - ib-19.3.Datatype-Primitief-Boolean.DT
+draft: false
+---
+
+> Meer informatie over [[4. Deeltaken/3. Processen/4. Implementatieproces/2. Bouwen softwaresysteem/Boolean/1. Boolean|Boolean]]
+
+## Opdracht 1
+Maak een boolean aan en print het resultaat.
+
+### Specificaties
+- Creëer een `bool` genaamd `IsWaar`.
+- Print de waarde van de `IsWaar uit in de console.
+
+### Verwachte output:
+Als `IsWaar = true`, komt in de console:
+```
+true
+```
+Als `IsWaar = false`, komt in de console:
+```
+false
+```
+
+### Nu jij:
+``` csharp runner
+using System;
+
+Console.WriteLine();
+``` 
+
+> [!info]- Mogelijke uitwerking
+> ``` csharp
+> using System;
+>
+>bool IsWaar = true;
+>Console.WriteLine(IsWaar);
+> ```
+
+---
+## Opdracht 2
+Draai het resultaat om van een boolean.
+
+### Specificaties
+- Neem de code over van opdracht 1.
+- Draai de gedeclareerde waarde om.
+
+### Verwachte output:
+Als `IsWaar = true`, komt in de console:
+```
+false
+```
+Als `IsWaar = false`, komt in de console:
+```
+true
+```
+
+### Nu jij:
+``` csharp runner
+using System;
+
+Console.WriteLine();
+``` 
+
+> [!info]- Mogelijke uitwerking
+> ``` csharp
+> using System;
+>
+>bool IsWaar != true;
+>Console.WriteLine(IsWaar);
+> ```
+
+---
+
+## Opdracht 3
+Maak gebruik van een expressie in een while-loop.
+
+Indien je nog niet bekend bent met een While-loop, dan kan je eerst kijken naar de While-loop.
+Indien je nog niet bekend bent met een interger, dan kan je eerst kijken naar de Interger.
+
+### Specificaties
+- Maak een variable `CountDown` aan
+- Zolang de waarde van `CountDown` een hogere waarde heeft dan 0 moet er 1 van afgehaald worden. En wordt de huidige waarde geprint.
+- Als `CountDown` een waarde heeft van 0, toon "Go!"
+
+### Verwachte output:
+Als `CountDown = 5`, komt in de console:
+```
+5
+```
+Als `CountDown = 0`, komt in de console:
+```
+Go!
+```
+
```

Comment:
Opmerking: de tussenliggende output (van 4,3,2,1) wordt niet geprint. Is dat zoals bedoeld?